morph/storage000-hardware.nix

```diff
@@ -0,0 +1,37 @@
+# Do not modify this file!  It was generated by ‘nixos-generate-config’
+# and may be overwritten by future invocations.  Please make changes
+# to /etc/nixos/configuration.nix instead.
+{ config, lib, pkgs, ... }:
+
+{
+  imports =
+    [ <nixpkgs/nixos/modules/installer/scan/not-detected.nix>
+    ];
+
+  boot.initrd.availableKernelModules = [ "ahci" "xhci_pci" "ehci_pci" "megaraid_sas" "usbhid" "sd_mod" ];
+  boot.initrd.kernelModules = [ ];
+  boot.kernelModules = [ "kvm-intel" ];
+  boot.extraModulePackages = [ ];
+
+  fileSystems."/" =
+    { device = "/dev/disk/by-uuid/ccabaa39-d888-467e-b8d9-75b5790a91aa";
```

Comment:
should the UUID of the disks be hard coded?

Reply:
I think it should.  This is basically equivalent to fstab.  By UUID is easiest way to unambiguously identify a disk (sdX will change if the hardware reports disks in a different order, such as if physical connections are moved around or even potentially on a firmware upgrade).  If the disk with the root partition actually dies and gets replaced we will have to change this configuration... but as things currently stand we will also have to reinstall NixOS overtop of Debian so we might just regenerate this file anyway.